pkg/cmd/label/edit.go

```diff
@@ -0,0 +1,104 @@
+package label
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/ghrepo"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type editOptions struct {
+	BaseRepo   func() (ghrepo.Interface, error)
+	HttpClient func() (*http.Client, error)
+	IO         *iostreams.IOStreams
+
+	Color       string
+	Description string
+	Name        string
+	NewName     string
+}
+
+func newCmdEdit(f *cmdutil.Factory, runF func(*editOptions) error) *cobra.Command {
+	opts := editOptions{
+		HttpClient: f.HttpClient,
+		IO:         f.IOStreams,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "edit <name>",
+		Short: "Edit a label",
+		Long: heredoc.Docf(`
+			Update a label on GitHub.
+
+			A label can be renamed using the %[1]s--name%[1]s flag.
+
+			The label color needs to be 6 character hex value.
+		`, "`"),
+		Example: heredoc.Doc(`
+			# update the color of the bug label
+			$ gh label edit bug --color FF0000
+
+			# rename and edit the description of the bug label
+			$ gh label edit bug --name big-bug --description "Bigger than normal bug"
+	  `),
+		Args: cmdutil.ExactArgs(1, "cannot update label: name argument required"),
+		RunE: func(c *cobra.Command, args []string) error {
+			opts.BaseRepo = f.BaseRepo
+			opts.Name = args[0]
+			opts.Color = strings.TrimPrefix(opts.Color, "#")
+			if opts.Description == "" &&
```

Comment:
I want to explicitly set `Description` to `""` since many labels are self-explanatory, and Github itself allows that

(current workaround: `" "`)